spec/abstractops.html

```diff
@@ -686,16 +686,16 @@ <h1>FormatSecondsStringPart ( _second_, _millisecond_, _microsecond_, _nanosecon
       1. Assert: _second_, _millisecond_, _microsecond_, and _nanosecond_ are integers.
       1. If _precision_ is *"minute"*, return *""*.
       1. Let _secondsString_ be the string-concatenation of the code unit 0x003A (COLON) and ToZeroPaddedDecimalString(_second_, 2).
-      1. Let _fraction_ be _millisecond_ &times; 10<sup>6</sup> + _microsecond_ &times; 10<sup>3</sup> + _nanosecond_.
+      1. Let _fractionNanoseconds_ be _millisecond_ &times; 10<sup>6</sup> + _microsecond_ &times; 10<sup>3</sup> + _nanosecond_.
```

Comment:
Suggested renames (which might be overridden per https://github.com/tc39/proposal-temporal/pull/2607/files#r1264712212 anyway, so :shrug:):
```suggestion
      1. Let _subSecondNanoseconds_ be _millisecond_ &times; 10<sup>6</sup> + _microsecond_ &times; 10<sup>3</sup> + _nanosecond_.
      1. Let _fractionDigits_ be ToZeroPaddedDecimalString(_subSecondNanoseconds_, 9).
```